yt/data_objects/construction_data_containers.py

```diff
@@ -2437,6 +2437,8 @@ def export_ply(
         color_field=None,
         color_map=None,
         color_log=True,
+        color_field_max=None,
+        color_field_min=None,
```

Comment:
new arguments cannot be added in arbitrary places, you need to put them at the end of the signature, and preferably behind a `*,` (to make them keyword only). See https://github.com/yt-project/yt/issues/3992 for a more in-depth explanation.

Reply:
ok, I see now.  I'm new to the `*,` so some questions.  Should all keyword arguments be after the `*,`, i.e. should I add this even earlier in the signature (I'm guessing these are calls that have not yet been migrated to this new format)

Reply:
In an ideal world, yes ! Unfortunately we cannot do that without breaking someone's code (if anyone is passing these arguments positionally, making them keyword-only after the fact will break them).

Reply:
ok, I think I understand this now